examples/core/service/connection_main.py

```diff
@@ -0,0 +1,22 @@
+import asyncio
```

Comment:
Maybe put the whole file directly in examples/ folder as done in #121

Reply:
yes I'm waiting for that PR merged and pull it to this one